clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt

```diff
@@ -59,7 +59,7 @@ class OsvServiceFunTest : StringSpec({
 
         result.shouldBeSuccess { actualData ->
             val expectedData = OsvApiClient.JSON.decodeFromString<List<Vulnerability>>(expectedResult)
-            actualData.patchModifiedTimes() shouldBe expectedData.patchModifiedTimes()
+            actualData.patchModifiedTimes() shouldContainExactlyInAnyOrder expectedData.patchModifiedTimes()
```

Comment:
Just a question for my understanding: This is not related to current test failures, is it?

Reply:
Not this time (as I'm also changing orders in the new expected results), but I've seen failure before where just having this would have fixed it.